views/index.html

```diff
@@ -20,9 +20,20 @@
     <div class="container">
       <div class="row">
         <div class="col-md-8 col-md-offset-2">
-          <h1>Under Construction</h1>
+          <h1>Adam's personal API Project</h1>
           <h4><a href="/api">Read My API Documentation</a></h4>
         </div>
+
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2">
+          <h1><a href="/projects">Info about Projects</a></h1>
```

Comment:
It's odd to create a link that does not work at all--why did you add this link rather than working on actually rendering project info on a page?